### PR TITLE
(feat)add initial implementation of ph board

### PR DESCRIPTION
**Pull request by ranjib**
@robertobuti @theatrus review please

### CODE REVIEW

File: ph_board/channel.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	bus  i2c.Bus
+	addr byte
+}
+
+func (c *channel) Name() string {
+	return chName
+}
+
+func (c *channel) Read() (float64, error) {
+	if err := c.bus.WriteBytes(c.addr, []byte{0x10}); err != nil {
```
[the remark this answers is not in the document]
Comment (ranjib) [reply ph_board/channel.go]: The logic implemented here is an almost direct port of the sample c program. If I read that correctly, we need it only if we are doing high frequency reads (90 SPS). May be I am getting this wrong.
I'll go through the datasheet, but I doubt I'll understand whole lot.

[the remark this answers is not in the document]
Comment (ranjib) [reply ph_board/channel.go]: Ok, found another good resource, let me go through this as well : http://openlabtools.eng.cam.ac.uk/Resources/Datalog/RPi_ADS1115/

I am surprised embd or periph does not have a go driver that we can take a look for reference.

[the remark this answers is not in the document]
Comment (theatrus) [reply ph_board/channel.go]: Looks like I'll ignore this command (0x10), so thats fine.

File: ph_board/channel.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		return -1, err
+	}
+	buf, err := c.bus.ReadBytes(c.addr, 2)
```
Comment (theatrus) [inline ph_board/channel.go]: We may want to be explicit we want to read register 0 here.

**Comment by ranjib** [comment]
ph Conversion logic:
```sh
phValue= adcValue*5.0/1024/6; //convert the analog into millivolt
phValue=3.5*phValue+Offset;             //convert the millivolt into pH value
```

**Comment by theatrus** [comment]
Any value to millivolts instead of just calibrating on ADC counts directly? In the end it won't matter.

(The Pico board ADC uses a 2.5V reference, the reference is halved to drive the common signal)

File: ph_board/channel.go (hunk 2 above)
Comment (ranjib) [reply ph_board/channel.go]: 👍

**Pull request merged**